.github/workflows/helmsman_esignet.yml

```
@@ -92,7 +101,21 @@ jobs:
             echo "HELMSMAN_MODE=apply" >> $GITHUB_ENV
           else
             echo "HELMSMAN_MODE=${{ github.event.inputs.mode }}" >> $GITHUB_ENV
-          fi        
+          fi
+
+      - name: Set Profile
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:10:28: Double quote to prevent globbing and word splitting

(shellcheck)